templates/items/parts/item-activation.hbs

```diff
@@ -160,4 +160,23 @@
         </select>
     </div>
 </div>
+
+{{!-- Prompt Configuration --}}
+<div class="form-group">
+    {{#if (eq item.type "consumable")}}
+    <label class="checkbox">
+        <input type="checkbox" name="system.uses.autoDestroy" {{checked system.uses.autoDestroy}}> {{ localize "DND5E.ItemDestroyEmpty" }}
```

Comment:
Suggestion for a tooltip: "If this item has limited uses, it will be destroyed when those uses reaches 0. Otherwise it will be destroyed when used."